paymentsheet/src/test/java/com/stripe/android/paymentsheet/flowcontroller/FlowControllerConfigurationHandlerTest.kt

```diff
@@ -282,6 +303,48 @@ class FlowControllerConfigurationHandlerTest {
             assertThat(onInitCallbacks).isEqualTo(0)
         }
 
+    @OptIn(ExperimentalPaymentSheetDecouplingApi::class)
+    @Test
+    fun `Cancels current configure job if new call to configure comes in`() = runTest {
```

Comment:
We should also write a test for ensuring that the configure is cancelled if the viewModelScope is cancelled.